lib/containers/timeline-items/commit-container.js

```diff
@@ -9,15 +9,50 @@ export class Commit extends React.Component {
     item: PropTypes.object.isRequired,
   }
 
+  authoredByCommitter(commit) {
+    if (commit.authoredByCommitter) {
+      return true;
+    }
+    // If you commit on GitHub online the committer details would be:
+    //
+    //    name: "GitHub"
+    //    email: "noreply@github.com"
+    //    user: null
+    //
+    if (commit.committer.email === 'noreply@github.com') {
+      return true;
+    }
+    if (commit.committer.name === 'GitHub' && commit.committer.user === null) {
+      return true;
+    }
```

Comment:
Nice! I actually had no clue this was how it worked.

I dug into the GItHub.com source code a bit and found that the committer name will be `"GitHub Enterprise"` if we're running on GitHub enterprise. The noreply address is also based off the domain of the Enterprise install. I'm going to see if I can get a field added called `authoredViaWeb` or something that matches the internal check we do, but until then this works great. 👍